test/e2e/cluster/add_remove_cluster.go

```diff
@@ -38,8 +38,8 @@ var _ = PDescribe("[expansion] Test expanding/shrinking an existing cluster flee
 		By(fmt.Sprintf("Found submariner engine pod %q on %q", enginePod.Name, clusterCName))
 
 		By(fmt.Sprintf("Checking connectivity between clusters"))
-		dataplane.RunConnectivityTest(f, false, framework.GatewayNode, framework.GatewayNode, framework.ClusterC, framework.ClusterB)
-		dataplane.RunConnectivityTest(f, true, framework.NonGatewayNode, framework.NonGatewayNode, framework.ClusterC, framework.ClusterA)
+		dataplane.RunConnectivityTest(f, false, false, framework.GatewayNode, framework.GatewayNode, framework.ClusterC, framework.ClusterB)
```

Comment:
I suggest to turn those false/trues in to constants to make it easier to read.

const hostNetworking = true
const podNetworking = false

or something of that sort..

now it's becoming hard to read/understand what each true/false is about.